README.md

```diff
@@ -92,7 +92,7 @@ pip install mxnet-cu92mkl
 **Step 3:** Install or Upgrade MMS as follows:
 
 ```bash
-pip install -U mxnet-model-server
```

Comment:
I don't think we would need this. We can add a section to get the latest branch with "--pre". This could create a churn just before the release.

Reply:
If anyone read this README today, they will run into troubles.

Reply:
The release is very close by, do you see that this churn of docs rework is required?

Reply:
There's not much churn here. The docs should always be functional and accurate. Let's add --pre for the time being and point people to the 0.4 docs if they want what's in the current pip package.

Reply:
I concur that having --pre is better than not having it until public release. Does this require changes in a lot of places (that is what I can see as a churn). If it is not in a lot of places - I suggest to add --pre. We will remove it once we go public.

Reply:
If we go with a full scale change, we will need to revisit all the example docs and docker files. If we decide that this is the only file/commit, then it might be as simple as reverting a single commit.

Reply:
Reverts are messy.
Let's make the change, then the day of launch, we can add another commit that fixes it. We'll probably have minor updates to make at that point anyway.